JSDemos/menuMeta.json

```diff
@@ -522,9 +522,8 @@
             "DemoType": "Web"
           },
           {
-            "Title": "Multi-Row Headers (Bands)",
+            "Title": "Multi-Level Headers (Bands)",
             "Name": "MultiRowHeadersBands",
-            "DocUrl": "Guide/Widgets/DataGrid/Columns/Column_Types/Band_Columns/",
```

Comment:
We better remove `DocUrl`'s from other demos from this section, too.

Reply:
Should "Name": "MultiRowHeadersBands", be changed to "Name": "MultiLevelHeadersBands"?

Reply:
No, "Name" points at a folder with the demo.